x-pack/legacy/plugins/integrations_manager/public/screens/home.tsx

```diff
@@ -4,29 +4,92 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React, { useState, useEffect } from 'react';
-import { EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
-import { getIntegrationsList } from '../data';
-import { IntegrationListGrid } from '../components/integration_list_grid';
-import { IntegrationList } from '../../common/types';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  // @ts-ignore (elastic/eui#1557) & (elastic/eui#1262) EuiImage is not exported yet
+  EuiImage,
+  EuiPage,
+  EuiPageBody,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { PLUGIN, STATUS_INSTALLED, STATUS_NOT_INSTALLED } from '../../common/constants';
+import { IntegrationsGroupedByStatus } from '../../common/types';
+import { IntegrationsGridByStatus } from '../components/integration_list_grid';
+import { getIntegrationsGroupedByStatus } from '../data';
+import { useBreadcrumbs, useLinks } from '../hooks';
 
 export function Home() {
-  const [list, setList] = useState<IntegrationList>([]);
+  const { toListView } = useLinks();
+  useBreadcrumbs([{ text: PLUGIN.TITLE, href: toListView() }]);
+
+  const [map, setMap] = useState<IntegrationsGroupedByStatus>({
+    [STATUS_INSTALLED]: [],
+    [STATUS_NOT_INSTALLED]: [],
+  });
 
   useEffect(() => {
-    getIntegrationsList().then(setList);
+    getIntegrationsGroupedByStatus().then(setMap);
   }, []);
 
   return (
-    <EuiPanel>
-      <EuiTitle>
-        <h1>Elastic Integrations Manager</h1>
-      </EuiTitle>
-      <EuiSpacer />
-      <EuiText>
-        <h3>Available Integrations</h3>
-      </EuiText>
-      <EuiSpacer />
-      {list ? <IntegrationListGrid list={list} /> : null}
-    </EuiPanel>
+    <>
+      <EuiPage style={{ borderBottom: '1px solid #D3DAE6', paddingBottom: '7px' }}>
```

Comment:
You're going to run into theming issues here by applying hard-coded hex values. We highly recommend using EUI variables which we usually apply through [SASS](https://elastic.github.io/eui/#/guidelines/sass). However, there are ways to also get these variables into JS as well. I'd work with your team to find out how they've been handling styling and EUI tokens. Perhaps @jasonrhodes can help you out here.

Reply:
For sure. It won't ship this way. I made a comment about this in https://github.com/elastic/kibana/pull/42260/commits/5526ee42631dc2e998db35aa08a283667d0d1a2d but could have noted in the description or code as well